tests/conftest.py

```diff
@@ -18,3 +18,21 @@ def test_data_2022_monthly_45km(rootdir):
     filepath = os.path.join(rootdir, filename)
     ds = xr.open_dataset(filepath)
     return ds
+
+
+@pytest.fixture
```

Comment:
I am wondering as to whether it is useful to include a short one-line comment as to roughly what each of these test datasets represent (maybe not in this script?) since we now have 3 test datasets, could be useful for something like:

test_dataset_Jan2015_LAcounty_45km_daily: daily data for LA county at 45 km for Jan 2015, useful for daily index testing
test_dataset_01Jan2015_LAcounty_45km_hourly: hourly data for LA county at 45 km for 1 Jan 2015, useful for hourly index testing

Reply:
Agreed, thanks!